soc/arm64/xenvm/soc.h

```diff
@@ -0,0 +1,4 @@
+#ifndef _SOC_H
```

Comment:
Still don't understand why do we need this header

Reply:
We need this header because we need FLASH_MAP which need this header

Reply:
I've seached for soc.h in soc/arm64
The result is the following:
(ranger) ➜  arm64 git:(xrun) ✗ fnd soc.h
./nxp_imx/mimx8m/pinctrl_soc.h
./nxp_imx/mimx9/pinctrl_soc.h
./bcm_vk/viper/soc.h

where directory contents are: 
arm  bcm_vk  CMakeLists.txt  intel_socfpga  Kconfig  nxp_imx  nxp_layerscape  qemu_cortex_a53  qemu_virt_arm64  rockchip  xenvm

As you can see - not all socs have this header. Does it mean that qemu_cotrex_a53 for example, does not support FLASH?
There should be another way.